src/parquet/column/levels-test.cc

```diff
@@ -0,0 +1,193 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include <gtest/gtest.h>
+
+#include "parquet/thrift/parquet_types.h"
+#include "parquet/column/levels.h"
+
+using std::string;
+
+namespace parquet_cpp {
+
+const int NUM_LEVELS = 1024;
+
+class TestLevels : public ::testing::Test {
+ public:
+  void SetUp() {
+    level_values_.resize(NUM_LEVELS);
+    output_levels_.resize(NUM_LEVELS);
+    rle_bytes_.resize(2*NUM_LEVELS);
+    bit_packed_bytes_.resize(2*NUM_LEVELS);
```

Comment:
Can you make `NUM_LEVELS` a parameter? e.g. moving these to a method `void Init(size_t num_levels)`